packages/admin-next/dashboard/src/hooks/table/query/use-user-invite-table-query.tsx

```diff
@@ -0,0 +1,32 @@
+import { useQueryParams } from "../../use-query-params"
+
+type UseUserInviteTableQueryProps = {
+  prefix?: string
+  pageSize?: number
+}
+
+export const useUserInviteTableQuery = ({
+  prefix,
+  pageSize = 20,
+}: UseUserInviteTableQueryProps) => {
+  const queryObject = useQueryParams(
+    ["offset", "q", "order", "created_at", "updated_at"],
+    prefix
+  )
+
+  const { offset, created_at, updated_at, q, order } = queryObject
+
+  const searchParams = {
+    limit: pageSize,
+    offset: offset ? Number(offset) : 0,
+    order,
+    // created_at: created_at ? JSON.parse(created_at) : undefined,
+    // updated_at: updated_at ? JSON.parse(updated_at) : undefined,
+    // q, // Re-enable when params are fixed
```

Comment:
question: Do we need a todo/ticket for this?

Reply:
I've added `q` param on invites and users list endpoints in [this](https://github.com/medusajs/medusa/pull/7070/commits/7677076bfc4ef11d58d80b2107d96e26f595613c) commit so should be resolved now.

Reply:
Nice, let's get this in then 💪